docs/presto-admin-commands.rst

```diff
@@ -489,14 +489,14 @@ server upgrade
 **************
 ::
 
-    presto-admin server upgrade local_package_path [local_config_dir]
+    presto-admin server upgrade new-rpm [local_config_dir]
 
-This command upgrades the Presto RPM on all of the nodes in the cluster to the RPM specified
-by ``local_package_path``, preserving the existing configuration on the cluster. The existing
+This command upgrades the Presto RPM on all of the nodes in the cluster to the RPM
+``new-rpm``, preserving the existing configuration on the cluster. The existing
```

Comment:
Maybe clarify if given path is relative to `presto-admin` binaries or it has to be absolute?